### ISSUE TITLE
Nametag bug task

### ISSUE THREAD

**Issue by Willtom999** [WalnutDev27/PureChat-WalnutMod#3]
there is a new bug the bug is that when you create the faculty it does not appear on the nametag when you disconnect and reconnect the bug is no longer there we see the faction on the nametag

**Comment by Willtom999** [comment]
https://youtu.be/dtPJRLxNd9I

**Comment by Willtom999** [comment]
@WalnutDev27

**Comment by WalnutDev27** [comment]
Sorry for the inactivity, i was bussy with other server, i have a quiestion, you have other plugin that can modify the NameTag, no??
If this is true, please try disble this plugin and restart the server.

**Comment by Willtom999** [comment]
no

**Comment by Willtom999** [comment]
@WalnutDev27

**Comment by Willtom999** [comment]
Is not possible

**Comment by WalnutDev27** [comment]
why?

**Comment by Willtom999** [comment]
Its comes from the plugin that you have not properly configured

**Comment by Willtom999** [comment]
Nametag faction

**Comment by Willtom999** [comment]
@WalnutDev27

**Comment by WalnutDev27** [comment]
Ok, if anything, put the modified version that was made exactly for this plugin

**Comment by Willtom999** [comment]
Plz fix problem

**Comment by Willtom999** [comment]
ok

**Comment by Willtom999** [comment]
or: Class _64FF00\PureChat\factions\SimpleFactions contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (_64FF00\PureChat\factions\FactionsInterface::getFactionDTR) in phar:///home/container/plugins/PureChat-WalnutMod.phar/src/_64FF00/PureChat/factions/SimpleFactions.php on line 11
[22:51:24] [Server thread/EMERGENCY]: An unrecoverable error has occurred and the server has crashed. Creating a crash dump
[22:51:24] [Server thread/EMERGENCY]: Please upload the "/home/container/crashdumps/Mon_Aug_2-22.51.24-UTC_2021.log" file to the Crash Archive and submit the link to the Bug Reporting page. Give as much info as you can. @WalnutDev27

**Comment by WalnutDev27** [comment]
ok, please use the update, for see if can be resolved.

**Comment by Willtom999** [comment]
![B717A797-A076-44A7-96A4-9C40C7DF1D72](https://user-images.githubusercontent.com/83521998/127934899-601529bb-58df-40b1-8890-d8fc4dcf4bf2.png)
![3C8E32EB-0269-4409-9100-C8009ECE8891](https://user-images.githubusercontent.com/83521998/127934902-8a5188d8-fc13-4e61-a443-cf9551314603.png)

**Comment by Willtom999** [comment]
Bruh @WalnutDev27

**Comment by Willtom999** [comment]
Possible fix problem? @WalnutDev27

**Comment by WalnutDev27** [comment]
so, so. Really i don't knos what happened with this section, but you can use the PureChat version that was maded for resolve the factions compatibility, can testing this https://poggit.pmmp.io/r/119566/PureChat_dev-2.phar.

**Comment by Willtom999** [comment]
did you take ayzrix's? @WalnutDev27

**Comment by WalnutDev27** [comment]
no, this is a version official that you can get in the github repo of the SimpleFactions

**Comment by Willtom999** [comment]
ok

**Comment by willtom6666** [comment]
Is plugin Is fix?

**Comment by Willtom999** [comment]
@WalnutDev27 ?

**Comment by WalnutDev27** [comment]
This plugin should work

**Comment by Willtom999** [comment]
? simplefaction does not work with nametag

**Comment by WalnutDev27** [comment]
No, this plugin isn't working with NameTags

**Comment by WalnutDev27** [comment]
This plugin is very bug

**Comment by Willtom999** [comment]
😢

**Comment by Willtom999** [comment]
well why does purechat walk like? @WalnutDev27
